### ISSUE TITLE
Submit all packages to Stackage

### ISSUE THREAD

**Issue by SmartHypercube** [IHaskell/IHaskell#1390]
I wonder is there any specific reason all those `ihaskell-*` packages are not submitted to Stackage?

In my workflow, I build and install a version of IHaskell locally, and also have multiple projects which need IHaskell. Each project has different dependencies, but they all have to be careful to use exactly the same versions for `ihaskell-*` packages (by including a rather long section in `stack.yaml`). It's difficult to maintain, and when I want IHaskell on a new machine, I also have to be careful to build and install the exact same version all my projects use. If all these packages' versions are locked in a Stackage snapshot, it will be far easier to simply use the same Stackage LTS version for all my projects.

**Comment by SmartHypercube** [comment]
I would like to know the opinions of current maintainers on this issue.

If you think it's a good idea but don't have time, I'm willing to help by submitting to Stackage with me as the maintainer and deal with Stackage-related issues.

**Comment by vaibhavsagar** [comment]
I agree it's a good idea to submit the updated `ihaskell-display` packages. I think the first step is to upload new versions to Hackage, I'll do that in the next couple of days (please feel free to ping me again if it isn't done by Monday).

**Comment by SmartHypercube** [comment]
I didn't notice that currently these packages are not routinely released and uploaded to Hackage. Please upload when you have time. If there are difficulties on this, I would like to help too.
